src/card/stories.js

```diff
@@ -0,0 +1,135 @@
+/* @flow */
+/*global module */
+import React from 'react';
+
+import {storiesOf} from '@storybook/react';
+import {withReadme} from 'storybook-readme';
+
+//$FlowFixMe
+import CardReadme from '../../rfcs/card-component.md';
+
+import {LightTheme} from '../themes';
+import {ThemeProvider} from '../styles';
+
+import BaseCard, {hasThumbnail} from './card';
+import {
+  Action as CardAction,
+  Body as CardBody,
+  Thumbnail as CardThumbnail,
+  Title as CardTitle,
+} from './styled-components';
+
+import {thumbnail as thumbnailImg, header as headerImg} from './images';
+
+import type {Props} from './card';
+
+const cardWidth = '328px';
+const placeholderTitle = 'Card Title Entry';
+const placeholderText =
+  'Proin ut dui sed metus pharetra hend rerit vel non mi. Nulla ornare faucibus ex, non facilisis nisl. Maecenas aliquet mauris ut tempus cursus. Etiam semper luctus sem ac blandit. ';
+
+const Card = (props: Props) => (
+  <BaseCard style={{width: cardWidth}} {...props} />
```

Comment:
How about passing style overrides explicitly through overrides for a specific presentational component?